src/core/LiterallyCanvas.coffee

```diff
@@ -313,7 +320,7 @@ module.exports = class LiterallyCanvas
     return unless shapes.length
     drawShapes = =>
       for shape in shapes
-        renderShapeToContext(ctx, shape, {retryCallback})
+        shape.draw(ctx, { retryCallback })
```

Comment:
Funny thing about that: if you read the top of `src/core/shapes.coffee`, you'll see this in `defineShape`:

```coffee
  Shape.prototype.draw = (ctx, retryCallback) ->
    renderShapeToContext(ctx, this, {retryCallback})
```

It's just a shim to support people who use undocumented behavior. :-) I removed it from the API a couple releases ago.